SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -792,17 +792,7 @@ void USpatialActorChannel::DynamicallyAttachSubobject(UObject* Object)
 
 	check(Info != nullptr);
 
-	// Check to see if we already have authority over the subobject to be added
-	if (NetDriver->StaticComponentView->HasAuthority(EntityId, Info->SchemaComponents[SCHEMA_Data]))
-	{
-		Sender->SendAddComponentForSubobject(this, Object, *Info, ReplicationBytesWritten);
-	}
-	else
-	{
-		// If we don't, modify the auth delegation to gain authority.
-		PendingDynamicSubobjects.Add(TWeakObjectPtr<UObject>(Object));
-		Sender->GainAuthorityThenAddComponent(this, Object, Info);
-	}
+	Sender->SendAddComponentForSubobject(this, Object, *Info, ReplicationBytesWritten);
```

Comment:
We don't need to reactively add component auth because it'll be included in the set